app/views/shared/_open_on_assistant_modal.html.erb

```diff
@@ -1,12 +1,17 @@
-<iframe id="js-hidden-iframe" src="about:blank" style="display:none"></iframe>
+<% unless disabled %>
+  <iframe id="js-hidden-iframe" src="about:blank" style="display:none"></iframe>
+<% end %>
 <div class="mt-3 pr-2">
   <details class="dropdown details-reset details-overlay d-inline-block">
-    <summary id="js-download-repos-btn" class="btn" aria-haspopup="true">
+    <summary <%=  disabled ? "" : "id=\"js-download-repos-btn\"" %> class="btn <%= disabled ? "disabled" : "" %>" aria-haspopup="true">
```

Comment:
The id of the `summary` has extra quotes around it with this (`"js-download-repos-btn"` instead of `js-download-repos-btn`)

Should be able to just drop the disabled check on the id since the user can't click if it's disabled anyways :

```suggestion
    <summary id="js-download-repos-btn" class="btn <%= disabled ? "disabled" : "" %>" aria-haspopup="true">
```

Reply:
The `id` of the summary does not have extra quotes around it:
```ruby
"id=\"js-download-repos-btn\""
``` 
☝️ is a string type return value. It would produce the string: `id="js-download-repos-btn"`

---
The reason for removing the `id` was because it causes a javascript `TypeError`:

![oct-17-2018 12-32-51](https://user-images.githubusercontent.com/11095731/47101724-e3e7f100-d208-11e8-8c8e-e180a2dd0b96.gif)
This gif is what happens when you apply the suggested change you gave ☝️

Reply:
Ah good catch! The quotes around the `id` are showing up as redundant on my inspector though and it's causing the modal to not reset after the app is launched when there are assignments:

What's happening right now:
![image](https://user-images.githubusercontent.com/7866060/47102629-d7609a00-d202-11e8-9b6b-f7479f515db8.gif)
What should happen:
![image](https://user-images.githubusercontent.com/7866060/47103016-e3009080-d203-11e8-975f-4ff7057941b4.gif)

Here's another fix, haven't seen the `undefined` error with this one:
```suggestion
    <summary id=<%=  disabled ? "" : "js-download-repos-btn" %> class="btn <%= disabled ? "disabled" : "" %>" aria-haspopup="true">
```

Reply:
Applied your suggestion ✅